docs/runbooks/SSPTemplateValidatorDown.md

```diff
@@ -1,30 +1,42 @@
 # SSPTemplateValidatorDown
+<!--apinnick, Nov 2022-->
 
 ## Meaning
 
-The Template Validator is responsible for validating the Virtual Machines do not violate the templates they are assigned to.
-This alert fires when all the Template Validator's pods are down.
+This alert fires when the Template Validator is down.
```

Comment:
@apinnick I think it should remain 
This alert fires when all the Template Validator's pods are down.